R/run.R

```diff
@@ -100,6 +131,60 @@ run_tutorial <- function(name = NULL, package = NULL, shiny_args = NULL) {
   })
 }
 
+validate_tutorial_path_is_dir <- function(path = NULL) {
+  if (is.null(path)) return(list(valid = FALSE))
+  if (!file.exists(path)) {
+    return(list(valid = FALSE, value = path))
+  }
+
+  if (!utils::file_test("-d", path)) {
+    stop.("If `name` is a path to a tutorial, it must be the path to a directory containing a signle tutorial.")
+  }
+
+  rmds <- list.files(path, pattern = "rmd$", ignore.case = TRUE)
```

Comment:
Add a `.` to signify the start of the extension

```suggestion
  rmds <- list.files(path, pattern = "\\.rmd$", ignore.case = TRUE)
```